templates/minimal/README.md

```diff
@@ -37,11 +37,25 @@
 - 🤏 This template is a minimal (in terms of complexity and the number of components)
 template for building a blockchain node.
 
+
+- 🔧 To simplify the process of using Zombienet with Omni Node, we've added a pre-configured development
+chain spec (dev_chain_spec.json) to the minimal template. Additionally, the zombienet-omni-node.toml file
+has been updated to include a valid chain spec path pointing to dev_chain_spec.json
```

Comment:
I would move this to the `#### Update `zombienet-omni-node.toml` with a valid chain spec path` section, right before the network spec sample where we instruct where the chain_spec_path updating should be done.

```suggestion
- 🔧 To simplify the process of starting the minimal-template with Zombienet and Omni Node, we've added a pre-configured development
chain spec (dev_chain_spec.json) to the minimal template. The `zombienet-omni-node.toml` file of this template points to it, but you can update it to an updated chain spec generated on your machine, as shown below:
```

Reply:
Unresolving this. Suggested to also move this to a more appropriate section. LMK what you think.

Reply:
I have made the appropriate changes @iulianbarbu.  FYI creating the script gh-issue

Reply:
https://github.com/paritytech/polkadot-sdk/issues/7267

Reply:
I guess you could assign it to me if you don't mind 😀